kokoro/ubuntu/periodic.sh

```diff
@@ -13,5 +13,22 @@ mvn -B clean install
 # https://stackoverflow.com/questions/3459928/running-a-specific-maven-plugin-goal-from-the-command-line-in-a-sub-module-of-a/26448447#26448447
 # https://stackoverflow.com/questions/11091311/maven-execjava-goal-on-a-multi-module-project
 cd dashboard
+
+# Step 1: Generate dashboards for the released BOMs with the latest DashboardMain class
+# For example https://storage.googleapis.com/cloud-opensource-java-dashboard/dashboard/target/1.1.0/dashboard.html
```

Comment:
I feel "dashboard/target" in the URL is unnecessary. I think it can be removed by updating Kokoro job config

```
strip_prefix: "github/cloud-opensource-java/"
```
to
```
strip_prefix: "github/cloud-opensource-java/dashboard/target"
```

@elharo What do you think?